leafmap/common.py

```
@@ -3950,6 +3950,16 @@ def geojson_to_gdf(in_geojson, encoding="utf-8", **kwargs: Any):
         with open(out_file, "w") as f:
             json.dump(in_geojson, f)
             in_geojson = out_file
+    elif isinstance(in_geojson, str) and in_geojson.startswith("http"):
+        try:
+            return gpd.read_file(in_geojson, encoding=encoding, **kwargs)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)